src/index.html

```diff
@@ -2,13 +2,41 @@
 <html lang="en">
 
 <head>
-<meta charset="UTF-8">
-<meta name="author" content="Ishaan" >
-<meta name="description" content="My Webpage">
+  We can thus add the following proper meta tags to our web page: 
 
 
-<title> Work Webpage </title>
-
+  ```html
+  
+  <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta name="description" content="Learn about job openings at Jovian and submit your application now">
+      <meta name="keywords" content="data science, software development, jobs">
+      <meta name="author" content="Jovian">
+      <meta name="robots" content="index, follow">
+      <meta property="og:title" content="Jovian Careers">
+      <meta property="og:description" content="Learn about job openings at Jovian and submit your application now">
+      <meta property="og:image" content="jovian_meta_image.png">
+      <meta name="twitter:title" content="Jovian Careers">
+      <meta name="twitter:description" content="Learn about job openings at Jovian and submit your application now">
+      <meta name="twitter:image" content="jovian_meta_image.png">
+      <title>Jovian Careers</title>
+      
+  
+  
+  ```
+  
+  

```

Comment:
Was no need for this description, kindly remove this.

Reply:
Onto it

Reply:
Fixed it